CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## [v0.0.59] - 2025-03-13
```

Comment:
this section should come after the main heading `# Changelog` before the most recent release